Source/common/SNTFileInfo.m

```diff
@@ -59,11 +59,13 @@ @implementation SNTFileInfo
 
 extern NSString *const NSURLQuarantinePropertiesKey WEAK_IMPORT_ATTRIBUTE;
 
-- (instancetype)initWithPath:(NSString *)path error:(NSError **)error {
+- (instancetype)initWithPath:(NSString *)path
+                resolvedPath:(BOOL)resolvedPath
+                       error:(NSError **)error {
```

Comment:
Instead of adding 2 new methods, why not just make this initWithResolvedPath:error and create a new initWIthPath:error: which just has the path resolving bit at the beginning?